### PR TITLE
fix: Responsive login/signup page

### PR DESCRIPTION
**Pull request by amitojsingh**
1. Responsive signup/ Login Page
2. Responsive Feedback Form. 

please test while reviewing.

### CODE REVIEW

File: src/components/gurfateh/Gurfateh.tsx [hunk 1]
```diff
@@ -5,15 +5,23 @@ export default function Gurfateh() {
...
+            {text('GURFATEH_PUNJABI')}
+          </h1>
           <p className='whitespace-pre-line'>{text('LOGIN_INFO')}</p>
         </div>
-      </div>
-      <div className="absolute -left-10 -bottom-1">
-        <img className='w-24 h-24 md:w-20 md:h-auto md:rounded-none rounded-full mx-auto' src={'/images/singhFateh.png'} alt="Singh Fateh" width={100} height={100} />
+        <div className='relative'>
+          <img
+            className='w-24 h-24 md:w-20 md:h-auto md:rounded-none rounded-full mx-auto absolute bottom-0 -left-10'
```
Comment (MySmilingTurban) [inline src/components/gurfateh/Gurfateh.tsx:18]: h-auto is stretching the gurfateh singh's picture, set w-auto too for that media query

File: src/components/modal/index.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ interface ModalProps {
 export const Modal = ({ children }: ModalProps) => {
   return (
     <div className={'absolute top-0 left-0 w-full h-full bg-darkBlue bg-opacity-80 z-10'}>
-      <section className='absolute w-1/3 border border-zinc-950 p-8 rounded-lg bg-white w-80% h-auto top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2'>
+      <section className='absolute w-4/5 xl:w-1/3 border border-zinc-950 p-8 rounded-lg bg-white w-80% h-auto top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2'>
```
Comment (MySmilingTurban) [inline src/components/modal/index.tsx:10]: the form should scrollable in itself as it is showing up as the attached image
login:signup-err2

File: src/styles/globals.scss [hunk 3]
```diff
@@ -18,7 +18,7 @@
 }
 
 html,body,#root{
-  height: 100%;
+height: 100%;
```
Comment (MySmilingTurban) [inline src/styles/globals.scss:21]: nit: fix indentation

File: src/components/gurfateh/Gurfateh.tsx (hunk 1 above)
Comment (MySmilingTurban) [reply src/components/gurfateh/Gurfateh.tsx:18]: Screenshot 2024-02-29 at 4 42 41 pm

**Review by MySmilingTurban — COMMENTED** [review]
We need to make some more change as here's how it looks on phone:

login:signup-err1

File: src/components/gurfateh/Gurfateh.tsx (hunk 1 above)
Comment (amitojsingh) [reply src/components/gurfateh/Gurfateh.tsx]: fixed veerji

File: src/components/modal/index.tsx (hunk 2 above)
Comment (amitojsingh) [reply src/components/modal/index.tsx]: fixed veer ji

File: src/styles/globals.scss (hunk 3 above)
Comment (amitojsingh) [reply src/styles/globals.scss]: fixed veer ji

**Comment by MySmilingTurban** [comment]
Please fix the sign up/sign in form, as you can see in the video it has a lot of white space underneath the form elements

https://github.com/KhalisFoundation/gurmukhi-web/assets/51584037/7c10de47-a93b-4c3f-8717-0b6c0b904922

**Comment by MySmilingTurban** [comment]
Also, please fix the responsiveness between width 1024 to 1280

https://github.com/KhalisFoundation/gurmukhi-web/assets/51584037/8206a87b-b771-455e-a2c2-c1ab7e5ef87c

**Comment by MySmilingTurban** [comment]
Plus, for screen with width less than 450 we need to make sure that Gurfateh Singh doesn't cover up the text

https://github.com/KhalisFoundation/gurmukhi-web/assets/51584037/0137cd70-1d46-4306-8118-003b03f442f7

**Comment by amitojsingh** [comment]
fixed @MySmilingTurban  veer ji please verify

File: src/components/gurfateh/Gurfateh.tsx [hunk 4]
```diff
@@ -5,15 +5,23 @@ export default function Gurfateh() {
...
+            {text('GURFATEH_PUNJABI')}
+          </h1>
           <p className='whitespace-pre-line'>{text('LOGIN_INFO')}</p>
         </div>
-      </div>
-      <div className="absolute -left-10 -bottom-1">
-        <img className='w-24 h-24 md:w-20 md:h-auto md:rounded-none rounded-full mx-auto' src={'/images/singhFateh.png'} alt="Singh Fateh" width={100} height={100} />
+        <div className='relative'>
+          <img
+            className='w-auto h-auto md:rounded-none rounded-full mx-auto absolute bottom-0 -left-10'
```
Comment (MySmilingTurban) [inline src/components/gurfateh/Gurfateh.tsx:18]: change bottom-0 to -bottom-1 to align gurfateh singh with text box

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
minor changes requested

File: src/components/gurfateh/Gurfateh.tsx (hunk 4 above)
Comment (amitojsingh) [reply src/components/gurfateh/Gurfateh.tsx]: fixed

File: src/components/signin/SignIn.tsx [hunk 5]
```diff
@@ -199,18 +199,20 @@ export default function SignIn() {
...
-              {text('OR').toUpperCase()}
-              <div className='w-1/4 h-[1px] bg-stone-950/[.2] ml-1'></div>
+            <div className='flex flex-col gap-4'>
+              <div className='flex justify-center items-center grey'>
+                <span className='w-1/4 h-[1px] bg-stone-950/[.2] mr-1'></span>
+                {text('OR').toUpperCase()}
+                <span className='w-1/4 h-[1px] bg-stone-950/[.2] ml-1'></span>
+              </div>
+              <button
+                className='w-full flex p-4 rounded-lg bg-transparent hover:bg-slate-100 text-black text-lg items-center justify-center gap-4 border border-gray-300'
```
[the remark this answers is not in the document]
Comment (amitojsingh) [reply src/components/signin/SignIn.tsx]: fixed

File: src/components/gurfateh/Gurfateh.tsx [hunk 6]
```diff
@@ -5,15 +5,23 @@ export default function Gurfateh() {
...
+            {text('GURFATEH_PUNJABI')}
+          </h1>
           <p className='whitespace-pre-line'>{text('LOGIN_INFO')}</p>
         </div>
-      </div>
-      <div className="absolute -left-10 -bottom-1">
-        <img className='w-24 h-24 md:w-20 md:h-auto md:rounded-none rounded-full mx-auto' src={'/images/singhFateh.png'} alt="Singh Fateh" width={100} height={100} />
+        <div className='relative'>
+          <img
+            className='w-auto h-auto md:rounded-none rounded-full mx-auto absolute -bottom-1 sm:bottom-0 -left-10'
```
Comment (MySmilingTurban) [inline src/components/gurfateh/Gurfateh.tsx:18]: why do we need sm:bottom-0 ??

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]

File: src/components/gurfateh/Gurfateh.tsx (hunk 6 above)
Comment (amitojsingh) [reply src/components/gurfateh/Gurfateh.tsx:18]: veer ji as you said to change -bottom-1 for width ~365 I change and for rest of the sizes I changed to bottom-0
As you can see I have updated Gurfateh code alot so this according to that I have added bottom-0

Comment (MySmilingTurban) [reply src/components/gurfateh/Gurfateh.tsx:18]: please change it all to -bottom-1 as it still feels off when in media query covered by sm

Screenshot 2024-03-02 at 11 08 38 am

Comment (amitojsingh) [reply src/components/gurfateh/Gurfateh.tsx]: done veer ji I changed all to bottom-0

Comment (MySmilingTurban) [reply src/components/gurfateh/Gurfateh.tsx]: veerji, -bottom-1 karna si saara

Comment (MySmilingTurban) [reply src/components/gurfateh/Gurfateh.tsx]: @amitojsingh

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Pull request merged**